projects/graph/graph.py

```diff
@@ -11,12 +11,21 @@ def add_vertex(self, vertex):
         """
         Add a vertex to the graph.
         """
-        pass  # TODO
+        #1
+        self.vertices[vertex] = set()
+
     def add_edge(self, v1, v2):
         """
```

Comment:
Nice and simple logic to follow Bryce 👍 , good job following the instructions and making this _directed_. If you wanted too how could you tweak this code to make it _undirected_?